config-overrides.js

```diff
@@ -2,7 +2,19 @@ const webpack = require('webpack')
 
 module.exports = function override(config) {
   const fallback = config.resolve.fallback || {}
-  Object.assign(fallback, {})
+  require('react-app-rewire-postcss')(config /*, options */)
```

Comment:
We don't have any postcss plugins or anything configured, so I figured leaving this blank was the right option.